servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -5711,7 +5724,7 @@ JWSDomain retrieveJWSDomain(final String domainName) {
             LOG.debug("retrieveJWSDomain: retrieving domain {}", domainName);
         }
 
-        AthenzDomain athenzDomain = getAthenzDomain(domainName, true, false);
+        AthenzDomain athenzDomain = getAthenzDomain(domainName, true);
```

Comment:
why was the master flag set to false removed here since it ends up calling the same method?